### ISSUE TITLE
Grep no longer works on text output

### ISSUE THREAD

**Issue by brainlid** [jeremyjh/dialyxir#242]
Using Elixir 1.7.3
Dialyxir 1.0.0 RC3

Unable to grep the output anymore.

```bash
$ mix dialyzer --format dialyzer | grep driver
```
Related to issue #241, I think the output of the data is no longer being passed out through stdout? Or whatever it was passed out as before.

Downgrading to Dialyxir 0.5.1, this works...

```bash
$ mix dialyzer | grep lib/core/driver
```

I'm working with lots of error output (coming into a large existing project), there is no way to grep the output to limit it to a sub-section. If there was a filter that could be applied like in ExUnit, then this might not be an issue.

Example:

```bash
$ mix dialyzer lib/core/driver
```

As it is, the error output scrolls off my bash terminal and I can't filter it with grep or save it to a file. My only other option is expand the terminal history it stores.

**Comment by jeremyjh** [comment]
Yes indeed. The pipe operator only sends `stdout` input by default. We changed to stderr; this is more in keeping with *nix conventions and lets you directly only errors somewhere, if you like, and leave out all the "info".  It should be mentioned in the README though.

You can get the old behavior with something like `mix dialyzer 2>&1 | grep opaque`.

**Comment by brainlid** [comment]
@jeremyjh Thanks for the explanation and the suggested workaround. I would suggest adding it to the README or some other appropriate place. Thanks!

**Comment by esambo** [comment]
Yes, it would be nice to document this in the README, as it's very handy to filter large output to a specific sub directory using sed or awk, e.g.
```sh
mix dialyzer 2>&1 | awk '/some_directory_name/,/______/'
mix dialyzer 2>&1 | sed -n -e '/some_directory_name/,/______/p'
```
